src/node_jsvmapi.h

```diff
@@ -188,34 +188,43 @@ NODE_EXTERN napi_status napi_coerce_to_string(napi_env e,
 NODE_EXTERN napi_status napi_get_prototype(napi_env e,
                                            napi_value object,
                                            napi_value* result);
-NODE_EXTERN napi_status napi_property_name(napi_env e,
-                                           const char* utf8name,
-                                           napi_propertyname* result);
 NODE_EXTERN napi_status napi_get_propertynames(napi_env e,
                                                napi_value object,
                                                napi_value* result);
 NODE_EXTERN napi_status napi_set_property(napi_env e,
-                                          napi_value object,
-                                          napi_propertyname name,
+                                          napi_value o,
+                                          napi_value k,
```

Comment:
Agreed. I changed these names to match what's in the cc file - they were previously out-of-sync - but did consider changing them all to be full words. Didn't initially do this because most of the API in this file use letters for parameter names like this. 

I will make a pass and change these to use full words.